src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1122,13 +1131,15 @@ function GlobeControls(camera, domElement, engine) {
     };
 
     // update object camera position
-    this.updateCameraTransformation = function updateCameraTransformation(controlState)
+    this.updateCameraTransformation = function updateCameraTransformation(controlState, preventUpdate)
     {
         const bkDamping = this.enableDamping;
         this.enableDamping = false;
         state = controlState || CONTROL_STATE.ORBIT;
         update();
-        updateCameraTargetOnGlobe.bind(this)();
+        if (preventUpdate === undefined || preventUpdate) {
```

Comment:
If preventUpdate is true, `updateCameraTargetOnGlobe` is called, which seems contradictory to me.

Did you want to 
```js
if (!preventUpdate) { // 
   // update
}
```

Or even better, do you know javascript [now has default parameters value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters).

I then suggest to have the following signature: 
```js
function updateCameraTransformation(controlState, updateCameraTarget = true)
```

 and simply use `if (updateCameraTarget)` here.